pkgs/graphs/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: graphs
+version: 2.3.0
+description: Graph algorithms that operate on graphs in any representation
+repository: https://github.com/dart-lang/tools/tree/main/pkgs/graphs
```

Comment:
Anytime we make changes to the pubspec file we really should rev the version. Generally, I'd say that we should merge this in unchanged, and then plan to come along later w/ a PR to the tools/ repo that updates this to the correct value and revs the pubspec version.

No concerns however w/o merging this in as is, we should just plan to rev the pubspec after the merge).